www/front_src/src/Resources/Details/Header.tsx

```diff
@@ -63,6 +98,16 @@ const HeaderContent = ({ details }: DetailsSectionProps): JSX.Element => {
           )}
         </Grid>
       </Grid>
+      <Grid item>
+        <IconButton
```

Comment:
It should be nice to display it only on hover

Reply:
Why ? I think we should always display actionable buttons in side panel's header. As we're displaying enable/disable icon